### PR TITLE
records: remove double text in records 233 and 200

### LINKED ISSUE #1928: Remove double text under Related items in record 233 and 200
http://opendata-dev.web.cern.ch/record/233 
http://opendata-dev.web.cern.ch/record/200
have under "Related items" two entries. This is correct. However, the text "The files created are:" should be there only once, now it is for both related records

### PR DESCRIPTION
**Pull request by hjhsalo**
* Replace text "The files created are:" with just "-" in favor of list
  like syntax.
  (closes #1928)

Signed-off-by: Harri Hirvonsalo <harri.hirvonsalo@cern.ch>

### CODE REVIEW

**Comment by hjhsalo** [comment]
Currently in opendata-dev.web.cern.ch:
remove_double_text_old

Changed to this in this PR:
remove_double_text_new

**Comment by tiborsimko** [comment]
@hjhsalo You can keep the description for the first element and remove the description field altogether for the second.  The record will be displayed more nicely that way...

**Comment by hjhsalo** [comment]
Updated

File: cernopendata/modules/fixtures/data/records/cms-tools-ana-Run2011A.json [hunk 1]
```diff
@@ -91,77 +91,83 @@
...
       "name": "Lassila-Perini, Kati"
     }
-  ], 
+  ],
   "collections": [
     "CMS-Tools"
-  ], 
-  "date_created": "2011", 
-  "date_published": "2016", 
+<<<<<<< HEAD
```
Comment (tiborsimko) [inline cernopendata/modules/fixtures/data/records/cms-tools-ana-Run2011A.json]: @hjhsalo Whoops! Merge conflicts left in the PR...

**Comment by ioannistsanaktsidis** [comment]
@hjhsalo please rebase and I will merge also this one.

**Comment by hjhsalo** [comment]
@ioannistsanaktsidis 
Rebased. Sorry for the delay.

**Pull request merged**